File: Programmers/코딩테스트_입문/GwanSoon/day 5/배열 뒤집기.js

function solution(num_list) {
    
  const newArray = [];
  let cnt = num_list.length - 1; //왜냐하면 [0]부터 시작하기 때문.
  while(cnt >= 0) {
      newArray.push(num_list[cnt]);
      
      cnt = cnt - 1
  }
  return newArray;
}

File: Programmers/코딩테스트_입문/GangHyeok/Day05/배열 뒤집기.java
class Solution {
    public int[] solution(int[] num_list) {
        int[] answer = {};
        answer = new int[num_list.length];
        
        for(int i = num_list.length - 1 ; i >=0 ;i--){
            answer[num_list.length - i-1] = num_list[i];
        }
        
        return answer;
    }
}